libraries/ESP8266WebServer/src/ESP8266WebServer.h

```diff
@@ -80,6 +81,9 @@ class ESP8266WebServerTemplate
   using ClientType = typename ServerType::ClientType;
   using RequestHandlerType = RequestHandler<ServerType>;
   using WebServerType = ESP8266WebServerTemplate<ServerType>;
+  using ContentType_f = std::function<String(const String&)>;
```

Comment:
Another anal thing, we're not using Hungarian notation on other types/usings elsewhere in the code.